drivers/filters/mountmgr/device.c

```diff
@@ -506,13 +506,13 @@ MountMgrNextDriveLetterWorker(IN PDEVICE_EXTENSION DeviceExtension,
     DeviceInformation->LetterAssigned =
     DriveLetterInfo->DriveLetterWasAssigned = TRUE;
 
-    /* Browse all the symlink to see if there's already a drive letter */
```

Comment:
"the" is legit here.

Reply:
Re-added. (Suggest/Fix the whole sentence, if not what you expect.)

Reply:
tbh I don't see why this comment needs changing.
Is this just changing stuff to get the linecount up?